### ISSUE TITLE
Issue with ICESat2VegR.ATL03_photons_attributes_dt() unable to extract all beams

### ISSUE THREAD

**Issue by chenbadan** [carlos-alberto-silva/ICESat2VegR#3]
ATL03_photons_attributes_dt() extract only the last beam ("gt3r") by default. In the beams column of the result, only "gt3r" is included
![pho_error](https://github.com/user-attachments/assets/2500d53c-bf38-4d70-a19b-8963b87e0725)

**Comment by caiohamamura** [comment]
Hello @chenbadan , thank you for submitting feedback for us.

Are you sure that this dataset have other beams? Have you ran `atl03_h5[[1]]$beams` to check? 

The default is all beams as you can check yourself within the source code: 

https://github.com/carlos-alberto-silva/ICESat2VegR/blob/master/R%2FATL03_photons_attributes_dt.R#L48

**Comment by chenbadan** [comment]
Yes, I made sure that there are other beams present for this dataset. As you can see, the final result contains only the data for "gt3r"
![Uploading 1.jpg…]()

**Comment by chenbadan** [comment]
![Uploading 1.jpg…]()

**Comment by caiohamamura** [comment]
What is the result for running `atl03_h5[[1]]$beams`?

**Comment by chenbadan** [comment]
It is the result :
> library(ICESat2VegR)

> setwd('C:\\Users\\xl\\Desktop\\ICESat-2')

> file_path <- '.\\yongan'

> atl03_files <- list.files(file_path, "ATL03.*h5", full.names = TRUE)

> atl03_h5 <- ATL03_read(atl03_files[[1]])

> atl03_h5$beams
[1] "gt1r" "gt2r" "gt1l" "gt3l" "gt2l" "gt3r"

> atl03_pho_attr<- ATL03_photons_attributes_dt(atl03_h5)
  |                                                                   |   0%gt1l
  |===========                                                        |  17%gt1r
  |======================                                             |  33%gt2l
  |==================================                                 |  50%gt2r
  |=============================================                      |  67%gt3l
  |========================================================           |  83%gt3r
  |===================================================================| 100%

> unique(atl03_pho_attr$beam)
[1] "gt3r"
